### ISSUE TITLE
Docs: Add responsive nav_js images

### ISSUE THREAD

**Issue by benkoshy** [ddnexus/pagy#422]
### Responsive Navs

Add in responsive nav_js images for the following extras:
- [x] bootstrap
- [ ] bulma
- [ ] foundation
- [ ] materialize 
- [ ] semantic 
- [ ] tailwind 
- [ ] uikit

**Comment by ddnexus** [comment]
I thought we might want to pass on this one, especially because after #470 it would probably be enough. However, having a specific image capturing the responsiveness for each CSS would not hurt.

**Comment by benkoshy** [comment]
Seeing it was a low priority, we cut the scope to use the bootstrap responsive nav image everywhere - even to demonstrate how pagy looks in non-bootstrap css frameworks. This is less than ideal, but serves the purpose of demonstrating how a responsive nav looks. Instead of taking  (~4 hours photo-shopping) an acceptable outcome was achieved in less than 15 minutes. Closing in favour of #473.
